### DISCUSSION TITLE
iOS Shortcut to Add Info about Feeding

### DISCUSSION THREAD

**Discussion by iosmit** [babybuddy/babybuddy#388] [Show and tell]
You can download the shortcut here: 

[Feed Baby](https://www.icloud.com/shortcuts/96e5c97113354a9fb11e6325f443ff26)

**Comment by cdubz** [comment]
Thanks for sharing! Someone also shared some over in #300. I'd love to find a way to share these more effectively if we can cobble together a full set...

**Reply by iosmit** [reply]
You have my permission to combine them if you so wish into a single page.

**Reply by cdubz** [reply]
I was going to do that but surprisingly GitHub discussions doesn't have a "merge" feature 🤷

**Reply by iosmit** [reply]
I would just create another discussion called iOS Shortcuts and anyone who wants to contribute can just share in the comments

**Reply by cdubz** [reply]
Yeah good thought. I'll just merge them manually.

**Comment by cdubz** [comment]
Ok I merged this in to #300 -- thanks, again, for contributing!
